packages/salesforcedx-vscode-core/package.json

```diff
@@ -107,6 +112,13 @@
           "name": "Metadata",
           "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
         }
+      ],
+      "conflicts": [
+        {
+          "id": "conflicts",
+          "name": "Conflicts",
+          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
```

Comment:
This means that the conflict view can only be seen/used by non-scratch orgs. I think we should change this condition to showing the icon/view when a dx project is open and the user has enabled conflict detection. Here's an example of how we get the setting value in package.json https://github.com/forcedotcom/salesforcedx-vscode/blob/f6d905a4d53ab7599ff94cd231c5e5228a046445/packages/salesforcedx-vscode-lightning/package.json#L112

Reply:
@lcampos What is the benefit for displaying it for a scratch org context?

Reply:
@rcoringrato-sfdc sorry I forgot some of the implementation details about the feature. This should only be exposed for non-scratch orgs and we should also check that the setting is enabled.